### DISCUSSION TITLE
Firebase Analytics enable/disable options for data collection, gdpr, cookie consent?

### DISCUSSION THREAD

**Discussion by p-young** [firebase/firebase-js-sdk#6272] [General]
Hi,

I was wondering if there's any solutions or suggestions around firebase analytics and complying with gdpr.

There doesn't seem to be a way to disable data collection or change the behavior at all, whereas the ios and android sdk's seem to have it.

There is link here to disabling firebase analytics for web but it's broken:

https://support.google.com/firebase/answer/9019185?hl=en#zippy=%2Cin-this-article

**Comment by hsubox76** [comment]
You're correct, I don't see an easy way to get to this reference page from there but this should work:

https://firebase.google.com/docs/reference/js/analytics.md#setanalyticscollectionenabled

It probably belongs on this page along with the iOS and Android samples. https://firebase.google.com/docs/analytics/configure-data-collection?platform=ios

You can create an issue logging this as a documentation bug, if you like.
